client/src/types/error.rs

```diff
@@ -226,6 +226,12 @@ impl From<EngineReadError> for SnapshotError {
                 "Unsupported version: expected {:?}, found {:?}.",
                 expected, found
             )),
+            EngineReadError::UnsupportedAssociatedData => {
+                SnapshotError::Engine("Unsupported snapshot associated data".into())
```

Comment:
Do you suggest changing only some specific messages to lower case or all of them? Error messages in stronghold were originally in capital case. Not sure if this is a place to change that. Also, @thibault-martinez said that they use error messages to handle some error cases, changing error message case might break correct matching of errors.

Reply:
> The error message given by the Display representation of an error type should be lowercase without trailing punctuation, and typically concise.

I would suggest changing all of them to follow this [wider Rust convention](https://rust-lang.github.io/api-guidelines/interoperability.html?highlight=lowerca#error-types-are-meaningful-and-well-behaved-c-good-err), if possible in a non-breaking manner, otherwise not breaking things takes priority.

Reply:
Sure, I need a confirmation from @thibault-martinez

Reply:
I'm not aware that this is possible. It seems very hard to do if you use thiserror macros to implement `Display` for your error, for instance.

Reply:
I'm not aware that this is possible. It seems very hard to do if you use thiserror macros to implement `Display` for your error, for instance.